src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -71,7 +73,23 @@ const Authenticate = (() => {
             $('#exp_date_2').datepicker('setDate', '2099-12-31');
         }
     };
-
+    const handleResidenceList = (residence_list) => {
+        const $residence = $('#residence');
+        if (residence_list.length > 0) {
+            const $options_with_disabled = $('<select/>');
+            residence_list.forEach((res) => {
+                $options_with_disabled.append(makeOption({
+                    text       : res.text,
+                    value      : res.value,
+                    is_disabled: res.disabled,
+                }));
+            });
+            $residence.html($options_with_disabled.html());
+            // BinarySocket.wait('website_status').then(response => handleWebsiteStatus(response.website_status, $residence));
+        } else {
+            $residence.setVisibility(1);
```

Comment:
Question: This should make the element visible when the residence list is empty, right?